src/test/java/entities/CommonUserTest.java

```diff
@@ -0,0 +1,74 @@
+package entities;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CommonUserTest {
+
+    private CommonUser user;
+
+    @BeforeEach
+    void setUp() {
+        HashMap<String, Double> portfolio = new HashMap<>();
+        HashMap<String, TransactionHistory> history = new HashMap<>();
+        Double balance = 9740.0;
+
+        portfolio.put("GOOGL", 3.0);
+
+        Stock googleStock = new Stock(120.0, "Google", "GOOGL");
+        List<Transaction> googleTransactions = new ArrayList<>();
+        googleTransactions.add(new BuyTransaction(3.0, new PricePoint(LocalDate.now(), 120.0)));
+
+        history.put("GOOGL", new TransactionHistory(googleStock, googleTransactions));
+
+        user = new CommonUser(portfolio, history, balance);
+    }
```

Comment:
I think that this setup is great and can be reused (after readjustments obviously) for the other entities too.